conda_build/index.py

```diff
@@ -419,7 +418,9 @@ def _filter_add_href(text, link, **kwargs):
 
 
 def _maybe_write(path, content, write_newline_end=False, content_is_binary=False):
-    temp_path = join(gettempdir(), str(uuid4()))
+    # Create the temp file next "path" so that we can use an atomic move, see
+    # https://github.com/conda/conda-build/issues/3833
+    temp_path = '%s.%s' % (path, uuid4())
```

Comment:
is that possible to reproduce the error using a test?
It would be interesting to have a test for it.

Reply:
I’m afraid that’s no easily possible. 

The error was that `path` and the `temp_path` were on two different file systems and `temp_file` was big enough that there could happen a read on `path` while `temp_path` was being copied into it.

Now, both files are always in the same folder and thus on the same filesystem.